### DISCUSSION TITLE
PHP-CS-Fixer is not considered a linter

### DISCUSSION THREAD

**Discussion by Jean85** [renovatebot/renovate#24978] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest stable, it self-updates

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I have `:automergeLinters` in my config, but the MR that tries to update PHP-CS-Fixer says that automerge is disabled by config. It's a linter, and it's not being considered as that.

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment, marked as answer]
Renovate does not have any magic ability to know every linter in the world, so the list is manually maintained. PRs welcome

https://github.com/renovatebot/renovate/blob/0da50b51604ec918afbd5981ba8c834e03366484/lib/config/presets/internal/default.ts#L43-L51

https://github.com/renovatebot/renovate/blob/0da50b51604ec918afbd5981ba8c834e03366484/lib/config/presets/internal/packages.ts#L83-L92

**Reply by HonkingGoose** [reply]
If you want to update the preset, here's some more information: [Renovate developer docs, Creating/editing Renovate presets](https://github.com/renovatebot/renovate/blob/main/docs/development/creating-editing-renovate-presets.md).
